cg_lims/EPPs/files/file_to_udf.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from cg_lims.exceptions import LimsError, MissingArtifactError
+from cg_lims.get.files import get_result_file
+from cg_lims import options
+from pathlib import Path
+from genologics.entities import Artifact
+import pandas as pd
+
+import logging
+import click
+import sys
+import math
+
+LOG = logging.getLogger(__name__)
+
+
+def make_well_dict(process, lims, input):
+    """Creates a well dict based on input_output_map
+    keys: well of input artifact
+    values: input/output artifact depending on the input flag
+    """
+
+    well_dict = {}
+    for inp, outp in process.input_output_maps:
+        if outp.get("output-generation-type") == "PerAllInputs":
+            continue
+        in_art = Artifact(lims, id=inp['limsid'])
+        out_art = Artifact(lims, id=outp['limsid'])
+        source_art = in_art if input == True else out_art
+        col, row = source_art.location[1].split(':')
+        well = col + row
+        well_dict[well] = out_art
+    return well_dict
+
+
+def set_udfs(well_field: str, value_field: str, udf: str, well_dict: dict, result_file: Path):
+    """Reads the csv and sets the value for each sample"""
+
+    failed_arts = False
+    passed_arts = 0
+    csv_reader = pd.read_csv(result_file, encoding='latin1')
+    data = csv_reader.transpose().to_dict()
+    for i, sample in data.items():
```

Comment:
```suggestion
    for _, sample in data.items():
```

this way you can send the unused variable into the void